plugins/UsersManager/lang/en.json

```diff
@@ -63,6 +64,11 @@
         "SuperUserAccessManagementMainDescription": "Super users have the highest permissions. They can perform all administrative tasks such as adding new websites to monitor, adding users, changing user permissions, activating and deactivating plugins and even installing new plugins from the Marketplace.",
         "TheLoginScreen": "The login screen",
         "ThereAreCurrentlyNRegisteredUsers": "There are currently %s registered users.",
+        "TokenAuth": "API Authentication Token",
+        "TokenRegenerateConfirm": "Changing the API authentication token will invalidate the users current token. If the user is currently logged in he may be required to login again. Do you really want to continue?",
```

Comment:
Let's get some grammar guidance here :D

According to what I understand from http://www.grammarbook.com/punctuation/apostro.asp we have a display of possession between the user and the token. So probably right to have "user's" here as we are only talking about a single user and his token.

(for self reference regarding "logged in, he may": http://www.thepunctuationguide.com/comma.html#complexsentences)

But I think someone with a deeper knowledge and mastery of english punctuation rules will eventually find any errors left :D